src/app/helpers/filters/filters.ts

```diff
@@ -0,0 +1,63 @@
+import { InnerFilter } from "@baw-api/baw-api.service";
+import { isInstantiated } from "@helpers/isInstantiated/isInstantiated";
+import { AbstractModel } from "@models/AbstractModel";
+
+/**
+ * Adds a new filter condition to an existing filter
+ *
+ * @param filter The current existing filter conditions
+ * @param additionalFilter A new condition to add to the existing filter
+ * @returns A new filter which satisfies the intersection between the two filters
+ */
+export function addFilter(
```

Comment:
suggested naming convention:

`filterAnd` and usage would be `filterAnd(filter, newFilter)`

Carry that convention onto your other helper methods